packages/fxa-auth-server/config/index.js

```diff
@@ -569,6 +569,54 @@ const conf = convict({
     env: 'DEVICE_NOTIFICATIONS_ENABLED',
     default: true
   },
+  subhub: {
+    enabled: {
+      doc: 'Indicates whether talking to the SubHub server is enabled',
+      format: Boolean,
+      default: false,
+      env: 'SUBHUB_ENABLED'
+    },
+    useStubs: {
+      doc: 'Indicates whether to use stub methods for SubHub instead of talking to the server',
+      format: Boolean,
+      default: false,
+      env: 'SUBHUB_USE_STUBS'
+    },
+    stubs: {
+      plans: {
+        doc: 'Stub data used for plans',
+        format: Array,
+        env: 'SUBHUB_STUB_PLANS',
+        default: []
+      }
+    },
+    url: {
+      doc: 'SubHub Server URL',
+      format: 'url',
+      default: 'https://subhub.services.mozilla.com/',
+      env: 'SUBHUB_URL'
+    },
+    key: {
+      doc: 'Authentication key to use when accessing SubHub server',
+      format: String,
+      default: 'Correct_Horse_Battery_Staple_1',
```

Comment:
Not sure if it's well-established enough to be considered a convention, but elsewhere we have used the literal string `YOU MUST CHANGE ME` for these. Also not sure if that actually does anything to reduce the chance of it not being set, but mentioned fwiw. If nothing else it's a clear signal for self-hosters.

Reply:
(although self-hosters won't need to mess with this one of course, so maybe it's a moot point)

Reply:
I just copied this directly from the defaults for pushbox, since I based the subhub module on it. But I can change it to `YOU MUST CHANGE ME` - doesn't much matter here since there's likely not going to be a public repo with a self-hostable subhub